cmd/login_cloud.go

```diff
@@ -14,37 +16,96 @@ var loginCloudCommand = &cobra.Command{
 	Short: "Authenticate with Load Impact",
 	Long: `Authenticate with Load Impact.
 
-This will set the default server used when just "-o cloud" is passed.`,
+This will set the default token used when just "k6 run -o cloud" is passed.`,
+
+	Example: `
+  # Show the stored token.
+  k6 login cloud -s
+
+  # Set up the token.
```

Comment:
Maybe "Persist the specified API token to configuration file on disk"?

Reply:
Maybe "Store a token"?